lab_2_tokenize_by_bpe/main.py

```diff
@@ -26,6 +38,20 @@ def collect_frequencies(
     :param end_of_word: a token that signifies the end of word
     :return: dictionary in the form of <preprocessed word: number of occurrences>
     """
+    if not isinstance(text, str) or not isinstance(end_of_word, str) or not (
+            isinstance(start_of_word, str) or start_of_word is None):
+        return None
+
+    dict_frequencies = {}
+
+    splitted_text = text.split()
+    for i in splitted_text:
```

Comment:
try to optimize - do not count doe not unique words

Reply:
done